### ISSUE TITLE
Need to add the h1 tag on archive, search and author pages.

### ISSUE THREAD

**Issue by pegazee** [ahmedkaludi/accelerated-mobile-pages#3323]
In swift theme, On archive, search & author page, the title was in the h3 tag. So, we need to make in the h1 tag and make sure that the design should not be a break. For further information, we can contact & discuss this with @ahmedkaludi 

Screenshot: https://monosnap.com/file/8NavznggKGjgGnEPDqWq915y5zGTo9

Note: I've checked in the swift theme, We need to check also on the other designs including the amp layout also.

**Comment by MohammedKaludi** [comment]
Do consult me before starting the work, the what @pegazee mentioned is partially correct solution.

**Comment by akashagrjsr** [comment]
Another Issue: https://secure.helpscout.net/conversation/895415821/72670/

**Comment by MohammedKaludi** [comment]
Still Incorrect heading.

**Comment by MohammedKaludi** [comment]
👍 ready to merge.

**Comment by MohammedKaludi** [comment]
pushed in 0.9.97.64
